.pullapprove.yml

```diff
@@ -41,3 +41,16 @@ groups:
           - "*.md"
         exclude:
           - "vendor/*"
+
+  protocol-changes:
+    required: 1
```

Comment:
`required: 1` feels a little slim to me - if it is architectural change then I would not be against increasing this .

Reply:
Oh yeah I've missed that actually. @jodh-intel could you please put `2` or `3` here ?
I think `2` is already the default for the agent repo, isn't it ?

Reply:
Ha - I put in `1` as I was expecting folk to fight against `>1` ;)

I''ve now changed it to `2`. The ideal in my mind would be `1` from the arch team and `1` from one of the other two teams. However, fwics, that would require a bit of extra bloat in the config as we'd need two "protocol-changes" groups - one just for the arch team and another for the other two teams.

Reply:
`2` is fine for now, we'll introduce protocol-changes group in a follow-up PR I guess